test.js

```diff
@@ -436,3 +436,447 @@ test('highlight()', function (t) {
 
   t.end()
 })
+
+// Light section
+
+var light = require('./light')
+var js = require('highlight.js/lib/languages/javascript')
+var as = require('highlight.js/lib/languages/applescript')
+var cp = require('highlight.js/lib/languages/cpp')
+var lt = require('highlight.js/lib/languages/latex')
+var su = require('highlight.js/lib/languages/subunit')
```

Comment:
please move these to the top, so when opening this file it’s immediately clear what’s loaded

Reply:
Should be good.